### PR TITLE
Fix Function test failed case

### PR DESCRIPTION
**Pull request by sharkconi**
Signed-off-by: Mark.Ma <mark.ma@emc.com>

### CODE REVIEW

**Comment by sharkconi** [comment]
@XiaowenJiang @xiar @fub2 @turtle-fly

File: test/functional/test_configuration_change.py [hunk 1]
```diff
@@ -68,7 +77,12 @@ def test_set_memory_capacity(self):
         self.conf["compute"]["memory"]["size"] = 1536
         with open("test.yml", "w") as yaml_file:
             yaml.dump(self.conf, yaml_file, default_flow_style=False)
-        qemu.start_qemu("test.yml")
```
Comment (fub2) [inline test/functional/test_configuration_change.py]: Then qemu could be really obsoleted.

**Comment by fub2** [comment]
tested on 2 very slow virtual machines. All functional tests passed! 

+1

**Comment by xiar** [comment]
:+1:

**Pull request merged**